preload.js

```diff
@@ -159,6 +161,37 @@ if (process.isMainFrame) {
         const y = e.clientY
         if (document.elementFromPoint(x, y) === this) this.#onfocus()
       })
+
+      Pear.messages({ type: 'pear/setMinimize' }, (message) => {
+        this.setCtrl({ minimize: message.data })
+      })
+
+      Pear.messages({ type: 'pear/setMaximize' }, (message) => {
+        this.setCtrl({ maximize: message.data })
+      })
+    }
+
+    async setCtrl ({ maximize = true, minimize = true }) {
+      if (maximize === true) {
+        const max = this.root.querySelector('#max')
+        max.style.display = 'inline'
+        if (isMac) await gui.ipc.setMaximizable({ id: gui.id, value: true })
+      }
+      if (maximize === false) {
```

Comment:
Don't think there's any need for the if-else construct here and the one below, so easier to maintain without them:

```
        const min = this.root.querySelector('#min')
        min.style.display = minimize ? 'inline' : 'none'
        if (isMac) await gui.ipc.setMinimizable({ id: gui.id, value: minimize })
```